Source/ACE/Entity/GenericObject.cs

```diff
@@ -15,46 +15,24 @@ public GenericObject(AceObject aceObject)
         {
         }
 
-        public GenericObject(ObjectGuid guid, AceObject aceObject)
-            : base(guid, aceObject)
-        {
-        }
+        ////public GenericObject(ObjectGuid guid, AceObject aceObject)
```

Comment:
I'd rather it echo out something to the effect of "Was asked to create a {WeenieType}, but don't know how" or something.  GenericObject is useless Object.